package.json

```diff
@@ -4,17 +4,19 @@
   "description": "Remix adapter and server runtime for Netlify",
   "main": "index.js",
   "scripts": {
-    "test": "vitest",
+    "build:packages": "pnpm run --filter '@netlify/*' build",
+    "build:packages:watch": "pnpm run --filter '@netlify/*' build:watch",
     "coverage": "vitest run --coverage",
+    "e2e": "pnpm exec playwright test",
```

Comment:
Perhaps `test:e2e`?

Reply:
That's my preference too, and I almost changed it! I wasn't sure if there was a naming convention at Netlify to use `e2e` already. 🤷🏼‍♂️

Reply:
I changed it 👍🏼